st2reactor/st2reactor/rules/config.py

```diff
@@ -44,16 +46,22 @@ def _register_common_opts():
 
 def _register_rules_engine_opts():
     logging_opts = [
-        cfg.StrOpt('logging', default='conf/logging.rulesengine.conf',
-                   help='Location of the logging configuration file.')
+        cfg.StrOpt(
+            'logging', default='conf/logging.rulesengine.conf',
+            help='Location of the logging configuration file.')
     ]
+
     CONF.register_opts(logging_opts, group='rulesengine')
 
     timer_opts = [
-        cfg.StrOpt('local_timezone', default='America/Los_Angeles',
-                   help='Timezone pertaining to the location where st2 is run.'),
-        cfg.BoolOpt('enable', default=True, help='Specify to enable Timer.')
+        cfg.StrOpt(
+            'local_timezone', default='America/Los_Angeles',
```

Comment:
Is `America/Los_Angeles` really a good default value? I would imagine UTC would be better, but I'm not really sure.